cypress/integration/models/migration/applicationinventory/application.ts

```diff
@@ -311,7 +311,7 @@ export class Application {
     tagExists(tagName: string): void {
         // Verifies if tag exists on application details -> Tags page
         this.applicationDetailsTab("Tags");
-        cy.get(applicationTag).should("contain", tagName);
+        cy.get(applicationTag, { timeout: 10 * SEC }).should("contain", tagName);
```

Comment:
For each tag it is clicking on application, checking tag and closing it . 

Instead it should click on application once, check all tags in a loop and close app at last.